### ISSUE TITLE
Sort Ignored domains by domain name alphabetically

### ISSUE THREAD

**Issue by Lagicrus** [AdguardTeam/AdGuardHome#6696]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

Finding if a domain name has already been added in the Ignored Domains section of the settings, UI is tricky when you get past, say, ten as they are in whatever order you entered them into the box and not sorted.

### Proposed solution

Sort the Ignored domains section alphabetically by the domain name to improve searchability.

**Comment by ainar-g** [comment]
I don't think we can do that.  These are currently matching rules following the usual [DNS filtering-rule syntax](https://adguard-dns.io/kb/general/dns-filtering-syntax/), which may include comments (for example, why domains matching these rules must be ignored), and we cannot sort those.
